ansible/roles/software/ckan/catalog/ckan-app/tasks/web.yml

```diff
@@ -1,13 +1,37 @@
 ---
+- name: Set some convenience facts about wsgi/gunicorn
+  set_fact:
+    gunicorn: "{{ uses_gunicorn | default(False) }}"
+    wsgi: "{{ not (uses_gunicorn | default(False)) }}"
```

Comment:
Since these are mutually exclusive, why not define `uses_gunicorn` in the role defaults, then the `when` conditions below become `uses_gunicorn` or `not uses_gunicorn`.

Otherwise, when I read the code below I'm thinking "do I need to write a test for when these are both enabled?". It should be clear from reading the tasks below that it's not possible.